python/MANOVA.qmd

```diff
@@ -0,0 +1,59 @@
+
+The **from_formula()** function is the recommended method to specify a model and simplifies testing without needing to manually configure the contrast matrices.
+
+**Example 39.6 Multivariate Analysis of Variance** from [SAS MANOVA User Guide](https://support.sas.com/documentation/cdl/en/statug/63033/HTML/default/viewer.htm#statug_glm_sect051.htm)
+
+This example employs multivariate analysis of variance (MANOVA) to measure differences in the chemical characteristics of ancient pottery found at four kiln sites in Great Britain. The data are from Tubb, Parker, and Nickless (1980), as reported in Hand et al. (1994).
+
+For each of 26 samples of pottery, the percentages of oxides of five metals are measured. The following statements create the data set and invoke the GLM procedure to perform a one-way MANOVA. Additionally, it is of interest to know whether the pottery from one site in Wales (Llanederyn) differs from the samples from other sites; a CONTRAST statement is used to test this hypothesis.
+
+
+```{python}
+
+import pandas as pd
+from statsmodels.multivariate.manova import MANOVA
+
+df= pd.read_csv("../data/manova1.csv")
+df.rename(columns={'al':'Al','fe':'Fe','mg':'Mg','ca ':'Ca','na':'Na'},inplace=True)
+
+manova = MANOVA.from_formula('Al + Fe + Mg + Ca + Na ~ site', data=df)
+result = manova.mv_test()
+print(result)
+```
+
+*Output*:
+
+                   Multivariate linear model
+===============================================================
+                                                               
+---------------------------------------------------------------
+        Intercept         Value  Num DF  Den DF F Value  Pr > F
+---------------------------------------------------------------
+           Wilks' lambda  0.0300 5.0000 18.0000 116.5838 0.0000
+          Pillai's trace  0.9700 5.0000 18.0000 116.5838 0.0000
+  Hotelling-Lawley trace 32.3844 5.0000 18.0000 116.5838 0.0000
+     Roy's greatest root 32.3844 5.0000 18.0000 116.5838 0.0000
+---------------------------------------------------------------
+                                                               
+---------------------------------------------------------------
+          site           Value   Num DF  Den DF F Value  Pr > F
+---------------------------------------------------------------
+          Wilks' lambda  0.0123 15.0000 50.0915  13.0885 0.0000
+         Pillai's trace  1.5539 15.0000 60.0000   4.2984 0.0000
+ Hotelling-Lawley trace 35.4388 15.0000 29.1304  40.5880 0.0000
+    Roy's greatest root 34.1611  5.0000 20.0000 136.6445 0.0000
+===============================================================
```

Comment:
Because the code gets run you don't need to have the outputs they will be included automatically. Quarto documents work very similarly to Jupyter notebooks

Reply:
Made necessary changes and submitted a pull request. Hope it is fine now